Rakefile

```
@@ -19,4 +20,11 @@ task :clean do
   Jekyll::Commands::Clean.process({})
 end
 
+# Test generated output has valid HTML and links.
+task :test do
+  sh "bundle exec jekyll build"

```

Isn't it better to do `Jekyll::Commands::Build.process(profile: true)` as on line 15?
```suggestion
  Jekyll::Commands::Build.process(profile: true)
```